config_panel.toml

```diff
@@ -18,6 +18,14 @@ name = "My Webapp configuration"
         visible = "with_sftp"
         help = "If a password already exist, leave blank and it will not be replaced."
 
+    [main.error]
+    name = "Html Custom error"
+
+        [main.error.errorfile]
+        ask = "HTML Path"
+        type = "string"
+        default = "/404.html"
```

Comment:
I'm confused because this setting value doesnt seem to have any effect whatsoever

Imho a boolean would be more suitable, something like : 

```
        [main.error.custom_404_file]
        ask = "Enable custom 404 file"
        type = "boolean"
        default = false
        help = "If you enable this, you can create a custom `404.html` file at the root of your app to act as the custom 404 error page"
```